Snowflake.Data/Core/EnvironmentVariables.cs

```diff
@@ -0,0 +1,12 @@
+// <copyright file="EnvironmentVariables.cs" company="Snowflake Inc">
+//         Copyright (c) 2019-2023 Snowflake Inc. All rights reserved.
+//  </copyright>
+
+namespace Snowflake.Data.Core
+{
+    public static class EnvironmentVariables
```

Comment:
The variables are not exclusive to SnowflaketomlConnectionBuilder, so they are outside of the class. Additionally, this file should not be constantly changing, so it would not generate so many conflicts.

Reply:
Moved to SnowflakeTomlConnectionBuilder

Reply:
but the class itself is now empty so it could be removed

Reply:
Done, removed!!!